flink-runtime/src/main/java/org/apache/flink/runtime/checkpoint/CompletedCheckpoint.java

```diff
@@ -320,6 +320,12 @@ void setDiscardCallback(@Nullable CompletedCheckpointStats.DiscardCallback disca
 
 	@Override
 	public String toString() {
-		return String.format("Checkpoint %d @ %d for %s", checkpointID, timestamp, job);
+		return String.format(
+			"%s %d @ %d for %s located at %s",
+			props.getCheckpointType(),
```

Comment:
Currently that would display:
"CHECKPOINT"
"SAVEPOINT"
"SYNC_SAVEPOINT"

I'm proposing to translate them into just "Checkpoint" and "Savepoint" to reduce the technicality. However, no strong feelings, so you can also convince me that it's better to leave as is.

Reply:
I do not have a strong opinion on this as well. But since you ask, let me have a try LOL :-)

The only concern is that we "might" have more types in the future
1. checkpoint or savepoint; 2. Synchronous or not; 3. Global or Individual; 4. unaligned or not (I mean it could be);

- It is a bit difficult to categorize the enum of CheckpointType to just "Checkpoint" and "Savepoint"
- It is also difficult to tell what level of technicality a user needs. For a user that makes an SYNC_SAVEPOINT, he/she probably has enough knowledge to understand what an SYNC_SAVEPOINT is. Or in other words, if we think SYNC_SAVEPOINT is not that understandable, we probably need to improve its expressiveness.

Reply:
Please let me know what do you think :-). I would be happy to change it.

Reply:
I guess having a bit technicality in the log should be okay. 

Another easy fix would be to override the `toString` in the checkpoint type and output it in more readable form.
"Checkpoint"
"Savepoint"
"Savepoint (sync)"

I also don't know if SYNC_SAVEPOINT would appear in materialized checkpoint at all - might be also some transient information that gets replaced by SAVEPOINT on recovery?